setup_src_bucket.sh

```diff
@@ -0,0 +1,8 @@
+S3_BUCKET_NAME=ai-art-tooter-src-bucket
```

Comment:
you can also create script with bucket name as input parameter.
Than you can call it with the bucket name and do not have this code twice

Reply:
changed